Swift.sublime-syntax

```diff
@@ -122,7 +134,7 @@ contexts:
         1: storage.type.function
         2: entity.name.function variable.function
       push: function_params
-    - match: '\b(deinit|init)\s*\('
+    - match: '\b(deinit|init(\?))\s*\('
```

Comment:
Double checking this - why do we want to match `init?` here?

Reply:
I've added a fix for those btw. We want to match both `init` and `init?` here. 

Because current version didn't match optional initializators anywhere and therefore they didn't appeared in the file symbol list also.